### PR TITLE
[Remove Stream/Optional usage] LPS-174299 Remove usages of Stream for module: portal-workflow-kaleo-metrics-integration

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:help

**Comment by W2jx** [comment]
ci:test:workflow

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-metrics-integration/src/main/java/com/liferay/portal/workflow/kaleo/metrics/integration/internal/helper/IndexerHelper.java [hunk 1]
```diff
@@ -420,20 +413,31 @@ public List<Assignment> toAssignments(
...
-			);
+			Map<Long, List<Long>> map = new HashMap<>();
+
+			for (KaleoTaskAssignmentInstance kaleoTaskAssignmentInstance :
+					kaleoTaskAssignmentInstances) {
+
+				long assigneeClassPK =
+					kaleoTaskAssignmentInstance.getAssigneeClassPK();
+
+				if (map.containsKey(assigneeClassPK)) {
```
Comment (dantewang) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-metrics-integration/src/main/java/com/liferay/portal/workflow/kaleo/metrics/integration/internal/helper/IndexerHelper.java:424]: @W2jx Could you try to use `Map.computeIfAbsent` to simplify this if-else?

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@W2jx Please resend, thanks.

**Pull request closed without merging**